common/pom.xml

```diff
@@ -18,6 +18,18 @@
         <jackson.modules.version>2.8.5</jackson.modules.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
The dependency management code allows use to use a single spring cloud release train version.